### ISSUE TITLE
Do old radars age out?

### ISSUE THREAD

**Issue by irons** [timburks/openradar#20]
I'm 80% sure I know the Google account under which I filed a bunch of old radars, but its "My Radars" section is empty. Searching for "irons" only turns up one hit, and "nathaniel" turns up the same plus one other (though it appears the originator field isn't searchable). Is there a possibility that My Radars wouldn't include results from ~2012?

I saved my google API key in QuickRadar, and it doesn't exactly match my Google account's current API key value, except for the last twelve digits, which match perfectly. This seems improbable.

**Comment by timburks** [comment]
No, I'm not aware of anything that would have caused your Radars to disappear. I'm working on an update that will include better search-by-user features. Thanks!
